nav2_mppi_controller/package.xml

```diff
@@ -29,6 +29,7 @@
   <depend>visualization_msgs</depend>
   <depend>eigen3_cmake_module</depend>
   <depend>eigen</depend>
+  <depend>benchmark</depend>
```

Comment:
?

Reply:
I'm just surprised since its not a dep I don't think in any other Nav2 package to pull in before.